composer.json

```diff
@@ -110,6 +110,7 @@
         "scssphp/scssphp": "v1.12.0",
         "setasign/fpdi": "^2.3.7",
         "setasign/tfpdf": "^1.33",
+        "shopware/commercial": "6.9.0",
```

Comment:
needs to be reverted!

Reply:
Oh damn, that is so annoying. Accidentally committed while resolving merge conflicts.

Reply:
Fixed